ansible/roles/sysprep/tasks/redhat.yml

```diff
@@ -44,6 +44,38 @@
 - name: Reset network interface IDs
   shell: sed -i '/^\(HWADDR\|UUID\)=/d' /etc/sysconfig/network-scripts/ifcfg-*
 
+
+- name: configure NetworkManager on RHEL 8.x
```

Comment:
This is not the correct place to do this, as this only runs when using Packer.
I _think_ we want another role where these, `Disable swap service and ensure it is masked` and `Ensure auditd is started`? are run.

cc @faiq @jkoelker WDYT?

Reply:
Good point about pre-provisioned. I was wondering what the `sysprep` boolean condition was for.  The `networking` role does not have OS specific tasks now. I can add this rhel 8 specific task there. Also I believe that this task should be removed once RHEL 8.x is able to take the latest NetworkManager rpm.

Reply:
Moved the task to `networking` role.